index.html

```diff
@@ -122,32 +122,32 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
-        <!-- ________ *Peter* Contributor card START ________  -->
+        <!-- ________ *Furkan* Contributor card START ________  -->
         <div class="card">
-          <p class="name">Chard Peter</p>
+          <p class="name">Furkan Murat</p>
           <p class="contact">
             <i class="fab fa-x-twitter"></i>
-            <a href="https://x.com/chardpeter1" target="_blank">@ChardPeter1</a>
+            <a href="https://twitter.com/Furkannnmurat" target="_blank">Furkannnmurat</a>
             <i class="fab fa-github"></i>
-            <a href="https://www.github.com/chardpeter" target="_blank">@chardpeter</a>
+            <a href="https://www.github.com" target="_blank">Your handle</a>
           </p>
-          <p class="about">I am an amazing honest man. Sweet and simpile. An introverty extrovert :D</p>
+          <p class="about">I just started the software business. I want to improve myself in this field.</p>
           <div class="resources">
             <p>3 Useful Dev Resources</p>
             <ul>
               <li>
-                <a href="https://freecodecamp.org" target="_blank" title="First Resource">Freecodecamp</a>
+                <a href="#" target="_blank" title="First Resource">Resource 1</a>
               </li>
               <li>
-                <a href="https://udemy.com" target="_blank" title="Second Resource">Udemy</a>
+                <a href="#" target="_blank" title="Second Resource">Resource 2</a>
```

Comment:
Please add a resource here (by replacing the "#" with the link to the resource and replacing "Resource 2" with the name of the resource.